roles/openshift_logging/templates/fluentd.j2

```diff
@@ -59,6 +59,11 @@ spec:
         - name: dockercfg
           mountPath: /etc/sysconfig/docker
           readOnly: true
+{% if openshift_logging_use_mux_client %}
```

Comment:
If you're adding the keys for the mux, here, where do you use them in fluentd collector?